pageserver/src/tenant/storage_layer/image_layer.rs

```diff
@@ -603,7 +602,10 @@ impl ImageLayerInner {
                     .as_slice()
                     .iter()
                     .filter_map(|(_, blob_meta)| {
-                        if blob_meta.key.is_rel_dir_key() || blob_meta.key == DBDIR_KEY {
+                        if blob_meta.key.is_rel_dir_key()
+                            || blob_meta.key == DBDIR_KEY
+                            || blob_meta.key.is_aux_file_key()
```

Comment:
Note that this checks against the aux v1 key which isn't used any more?

Reply:
May be I wrong, but I expect that it is check against V2.
At least `is_aux_file_key` is checking AUX_KEY_PREFIX (which is part off metadata key prefix).
And without this check I see `Oversized vectored read` warnings.

Reply:
sorry I missed that :(